app/frontend/wallet/shelley/shelley-transaction.ts

```diff
@@ -120,7 +120,7 @@ function cborizeTxInputs(inputs: TxInput[]): CborizedTxInput[] {
 function cborizeTxOutputTokenBundle(tokenBundle: TokenBundle): CborizedTxTokenBundle {
   const policyIdMap = new Map<Buffer, Map<Buffer, number>>()
   const tokenObject = groupTokenBundleByPolicyId(tokenBundle)
```

Comment:
I'd rename this to `orderedTokenBundle` and `groupTokenBundleByPolicyId` to `toOrderedTokenBundle` and perhaps even have a OrderedTokenBundle type to make it clear when such bundle is dealt with in the logic

Reply:
Great idea, fixed.